src/components/utils/rhf-inputs/select-inputs/countries-input.js

```diff
@@ -1,25 +1,10 @@
-import { useParameterState } from '../../../dialogs/parameters-dialog';
-import { PARAM_LANGUAGE } from '../../../../utils/config-params';
-import React, { useCallback, useMemo } from 'react';
-import { getComputedLanguage } from '../../../../utils/language';
+import React, { useMemo } from 'react';
 import { Chip } from '@mui/material';
 import { AutocompleteInput } from '@gridsuite/commons-ui';
+import { useCountriesListCB } from '../../dialog-utils';
 
 export const CountriesInput = ({ name, label }) => {
-    const [languageLocal] = useParameterState(PARAM_LANGUAGE);
-    const countriesListCB = useCallback(() => {
-        try {
-            return require('localized-countries')(
-                require('localized-countries/data/' +
-                    getComputedLanguage(languageLocal).substr(0, 2))
-            );
-        } catch (error) {
-            // fallback to english if no localised list found
-            return require('localized-countries')(
-                require('localized-countries/data/en')
-            );
-        }
-    }, [languageLocal]);
+    const countriesListCB = useCountriesListCB();
```

Comment:
In case of using src/components/utils/localized-countries-hook.js by duplication, do not need refactor this component. We can do it later..

Reply:
I don't think duplicating the code is good in the current context as we might merge the gridstudy and gridexplore soon and duplicating this code might lead to divergence. We'll update this when the merge is done.

Reply:
OK